src/src/models/OptionModel.ts

```diff
@@ -0,0 +1,5 @@
+export default interface OptionModel {
+    value: number;
+    name: string;
+    countForOption: number;
+};
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
}
```